### PR TITLE
Update refresh token patterns, tests

### PR DESCRIPTION
**Pull request by derekwheee**

### CODE REVIEW

File: lib/routes/auth/login.js [hunk 1]
```diff
@@ -16,21 +15,22 @@ module.exports = {
...
-            const results = await AuthService.login(username, password);
+            const results = await authService.login(username, password);
 
             if (!results.isBoom) {
                 h.state('_pallies', results.refreshToken, {
                     ttl: refreshTokenLifespan * 1000,
                     isSecure: !isDev,
-                    isHttpOnly: true
+                    isHttpOnly: true,
+                    path: '/login'
```
Comment (derekwheee) [inline lib/routes/auth/login.js]: I think this should be
```suggestion
                    path: '/reauthorize'
```

File: lib/routes/auth/resetPassword.js [hunk 2]
```diff
@@ -15,9 +13,9 @@ module.exports = {
             const AuthService = h.services().authService;
             const { oldPassword, userHash, forgotPasswordToken, newPassword } = request.payload;
 
-            const results = await AuthService.resetPassword(request.auth.credentials, oldPassword, userHash, forgotPasswordToken, newPassword);
+            await AuthService.resetPassword(request.auth.credentials, oldPassword, userHash, forgotPasswordToken, newPassword);
 
-            return new SuccessResult(results);
+            return h.response().code(204);
         }
```
Comment (marlonbrs) [inline lib/routes/auth/resetPassword.js:19]: If I understand correctly, if I need the data returned from calling `AuthService.resetPassword`, I'll use `/resetpassword2` route. If I don't and I only want to perform the action of resetting the password, I'll use this one?

**Review by marlonbrs — APPROVED** [review]
👀 ✅

File: lib/routes/auth/resetPassword.js (hunk 2 above)
Comment (derekwheee) [reply lib/routes/auth/resetPassword.js:19]: I think this is just some refactoring slop. `TokenService.resetPassword` doesn't return anything, so nothing would ever get return from there anyway. Just committed a change to clean that up :+1:

Comment (marlonbrs) [reply lib/routes/auth/resetPassword.js:19]: Ohh got it. Thanks for clarifying 🙏

**Pull request merged**